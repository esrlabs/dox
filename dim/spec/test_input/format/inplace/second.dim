document: abc
R1:
    type: heading_1
    text: Soso


