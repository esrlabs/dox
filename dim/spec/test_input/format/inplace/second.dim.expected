document: abc

R1:                         h1 Soso
